tldw_Server_API/app/core/Slides/style_packs/heritage_formal.css

```
@@ -0,0 +1,10 @@
+.reveal[data-style-pack="heritage_formal"] {
+  background-color: var(--surface, #fffbf5);
+  color: var(--text, #3f2a14);
+  font-family: "Baskerville", "Times New Roman", serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Baskerville" (font-family-name-quotes)

(font-family-name-quotes)